lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/core/expression/ExpressionsDsl.kt

```diff
@@ -69,6 +69,11 @@ public object ExpressionsDsl {
   public fun const(padding: PaddingValues.Absolute): Expression<PaddingValue> =
     Expression.ofPadding(padding)
 
+  /** Creates a literal expression for a list of strings. */
+  @JvmName("constStringList")
+  public fun const(list: List<String>): Expression<ListValue<StringValue>> =
+    Expression.ofList(listOf("literal", list)).cast()
```

Comment:
do we need one for numbers? we already have offset, dpoffset, and padding. Unsure if there's other lists of numbers around